src/standard-live-connect.ts

```diff
@@ -116,7 +116,7 @@ function _standardInitialization (liveConnectConfig: LiveConnectConfig, external
   }
 }
 
-export function StandardLiveConnect (liveConnectConfig: LiveConnectConfig, externalStorageHandler: ExternalStorageHandler, externalCallHandler: ExternalCallHandler, externalEventBus: EventBus): ILiveConnect {
+export function StandardLiveConnect (liveConnectConfig: LiveConnectConfig, externalStorageHandler: ExternalStorageHandler, externalCallHandler: ExternalCallHandler, externalEventBus?: EventBus): ILiveConnect {
```

Comment:
we can do that at the initializer and the minimalLC

Reply:
MinimalLiveConnect should already have it, but seems like I missed it for LiveConnect.

By the time we are calling the initialisers we always have an eventbus, so no need there

Reply:
you're right!